latest/index.html

```diff
@@ -638,10 +651,11 @@ <h3>Preview Metadata</h3>
       </section>
       <section>
       <h3>Preview Queries</h3>
-      <p>A preview service is queried by resolving the <a>preview URL</a> for an <a>entity</a>. The URL must resolve to an HTML document.
+      <p>A preview service is queried by resolving the URI template <code>/preview?id={id}</code> relative to the reconciliation endpoint, where <code>id</code> is subsituted by the <a>entity</a> identifier. The URL must resolve to an HTML document,
+        which MUST be viewable in an <code>iframe</code> HTML element whose dimensions are determined by the <a>preview metadata</a>.
```

Comment:
Shouldn't that be CAN instead?  I mean some clients don't necessarily need an iframe, it's up to them how to interact with data coming from a standardized api spec?

Reply:
That `MUST` does not force clients to embed the preview in an iframe, it only ensures that the server makes that possible. I would be fine with reformulating this to say something like "an HTML page which MUST be viewable in a viewport whose dimensions are…", if you want to avoid the term iframe…

Reply:
Yes would be better to rephrase like that I think.